api/src/pcapi/routes/backoffice/templates/components/links.html

```diff
@@ -108,3 +108,11 @@
        class="link-primary">{{ venue.siret }}</a>
   {% endif %}
 {% endmacro %}
+{% macro build_dms_adage_folder_link(venue) %}
+  {% if venue.dms_adage_folder_number %}
+    <a href="https://www.demarches-simplifiees.fr/procedures/{{ venue.dms_adage_folder_number.procedure }}/dossiers/{{ venue.dms_adage_folder_number.application }}"
+       target="_blank"
+       title="Visualiser le numéro de dossier demarche simplifié d'adage"
```

Comment:
Mais l'orthographe...
```suggestion
       title="Visualiser le numéro de dossier Démarches Simplifiées Adage"
```